service/resource/kmskeyv1/create_test.go

```diff
@@ -73,7 +69,7 @@ func Test_Resource_S3Object_newCreate(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		result, err := newResource.newCreateChange(context.TODO(), tc.obj, tc.currentState, tc.desiredState)
+		result, err := newResource.newCreateChange(context.TODO(), clusterTpo, tc.currentState, tc.desiredState)
```

Comment:
Minor but can this be customObject? TPOs are going soon :)

Reply:
\o/